docs/user_guide/notes.rst

```diff
@@ -0,0 +1,34 @@
+Notes for Administrators
+========================
+Make sure to completely read and understand the following before putting ``ngshare`` into production
+
+Race Condition
+--------------
+ngshare should NOT run concurrently, or there may be race conditions and data may be corrupted. For example, do not create multiple ngshare instances that share a same underlying database.
+
+Database Upgrade
+----------------
+ngshare checks database version every time it starts up. If database version is older than ngshare version, it performs schema and data migration. Normally this may only happen after ngshare is updated.
```

Comment:
```suggestion
ngshare checks database version every time it starts up. If database version is older than ngshare version, it performs schema and data migration. Normally, this may only happen after ngshare is updated.
```